packages/core/js-client/run-peer.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { KeyPair } from './src/keypair/index.js';
+import { RELAY } from './src/clientPeer/__test__/connection.js';
+import { ClientPeer } from './src/clientPeer/ClientPeer.js';
+import { WorkerLoader } from './src/marine/worker-script/workerLoader.js';
+import { WasmLoaderFromNpm } from './src/marine/deps-loader/node.js';
+import { MarineBackgroundRunner } from './src/marine/worker/index.js';
+import { multiaddr } from '@multiformats/multiaddr';
+
+// const key = '+cmeYlZKj+MfSa9dpHV+BmLPm6wq4inGlsPlQ1GvtPk=';
+// const keyBytes = toUint8Array(key);
+//
+// const privKey = await generateKeyPairFromSeed('Ed25519', keyBytes, 256);
+// const peer = new ClientPeer({}, new KeyPair(privKey, "12D3KooWBM3SdXWqGaawQDGQ6JprtwswEg3FWGvGhmgmMez1vRbR"));
+// await peer.start();
```

Comment:
These comments too. If any of the commented-out code is useful for debugging purposes for you in the future and you don't want to remove it - I would apprecitate it if you leave additional comment with some explanation on how to use it and when so other people could pottentially uncomment and use it as well when it's needed